### ISSUE TITLE
registerPropertyEditor throws undefined error in React 

### ISSUE THREAD

**Issue by kapso44** [unlayer/react-email-editor#117]
I have the below code 

`render() {
        this.editor.registerPropertyEditor({
            name: 'my_color_picker',
            Widget: this.MyColorPicker,
          });

        return (
            <div className="template-editor">
                <div id="template-editor">
                    <EmailEditor
                        ref={(editor) => (this.editor = editor)}`

and this throws error of undefined for registerPropertyEditor(). Image attached.

Please guide through this.
![registerPropertyIssueInReact](https://user-images.githubusercontent.com/13742885/80184519-5c164a80-8628-11ea-9a3b-fb5dca781ff6.png)

**Comment by adeelraza** [comment]
@kapso44 `registerPropertyEditor` has to be passed in `customJS`. Please check: https://docs.unlayer.com/docs/create-custom-tool#custom-property-editor

**Comment by kapso44** [comment]
> @kapso44 `registerPropertyEditor` has to be passed in `customJS`. Please check: https://docs.unlayer.com/docs/create-custom-tool#custom-property-editor

So you mean we have to define the color widget in the react and call that in customJs file. If that so how you will pass the `Widget:MyColorPicker` expression in customJs.

I have read the whole documentation and tried out all the possibilities, so please explain rightly how to use the widget as a react component. The documentation on react implementations are not clear enough.

**Comment by adeelraza** [comment]
@kapso44 Have you checked this example in our playground: https://codesandbox.io/embed/github/unlayer/playgrounds/tree/master/react/custom-tool-and-property-editor?from-embed

Here, we are defining a custom property editor in React.

**Comment by kapso44** [comment]
> @kapso44 Have you checked this example in our playground: https://codesandbox.io/embed/github/unlayer/playgrounds/tree/master/react/custom-tool-and-property-editor?from-embed
> 
> Here, we are defining a custom property editor in React.

Yeah, I have looked at the above example also. I have called the custom Js files using the above. But the example doesn't solve the mentioned issue.
